src/ProjectTemplates/Web.ProjectTemplates/content/RazorClassLibrary-CSharp/.template.config/template.json

```diff
@@ -22,12 +22,22 @@
   "preferNameDirectory": true,
   "sources": [
     {
-      "modifiers": [{
-        "condition": "(!SupportPagesAndViews)",
-        "exclude": [
-          "Areas/**"
-        ]
-      }]
+      "modifiers": [
+        {
+          "condition": "(!SupportPagesAndViews)",
+          "exclude": [
+            "Areas/**"
+          ]
+        },
+        {
+          "condition": "(SupportPagesAndViews)",
+          "exclude": [
+            "Component1.razor",
+            "ExampleJsInterop.cs",
+            "content/exampleJsInterop.js"
```

Comment:
The static content needs to be in a *wwwroot* folder instead of a *content* folder.

Also it looks like you're missing exclusions for the other content files (the image and css files). You if should just excluded everything under *wwwroot* support for pages and views is selected.

Reply:
I just copied their location from the BlazorLibrary template, why was it in content there but should be in wwwroot here? I'm also not sure I follow you about the exclusions, could you clarify/edit your comment?

Reply:
The old Blazor Class Library template used an older Blazor specific mechanism for handling the static assets. Now we have general support in ASP.NET Core 3.0 for handling static assets in class libraries, which uses the wwwroot folder convention.

The static content for the components version of this template template I believe consists of three files: a PNG image, a JS file, and a CSS file. The pages and views version of this template shouldn't have a wwwroot folder at all, but only the JS file is excluded here